### PR TITLE
Ports tg's winter boots 👞

### PR DESCRIPTION
**Pull request by FlattestGuitar**
Because they're cute.

They're in the loadout system now.

Workboots are there as well, since they don't do anything else than just look nice.

:cl: FlattestGeetar
tweak: Workboots are now in the loadout system.
rscadd: Ports winter boots from /tg/
/:cl:

I couldn't find any PR that added them, so I've no idea whose sprites these are.

### CODE REVIEW

**Comment by KasparoVy** [comment]
Since these are basically reskinned workboots, could you please use those sprites as a template and make toecut and Vox versions of these sprites?

**Comment by FlattestGuitar** [comment]
Filthy skrekbirbs breaking my PRs. 

Will do once I'm home.

**Comment by KasparoVy** [comment]
Thanks! As far as the toecut versions go I think they are just snipping off some pixels at the ends (though a look at the appropriate workboot toecut sprites would clear that up)

**Comment by FlattestGuitar** [comment]
🐦 📥 

Addressed.

**Comment by pinatacolada** [comment]
are those uggs?

**Comment by Spacemanspark** [comment]
About time we got these.

File: code/modules/client/preference/loadout/loadout_shoes.dm [hunk 1]
```diff
@@ -7,6 +7,14 @@
 	display_name = "sandals, wooden"
 	path = /obj/item/clothing/shoes/sandal
 
+/datum/gear/shoes/winterboots
+	display_name = "winter boots"
+	path = /obj/item/clothing/shoes/winterboots
```
Comment (FlattestGuitar) [inline code/modules/client/preference/loadout/loadout_shoes.dm]: Well, we already have the winter coat, which is way more powerful than the boots. Plus, my next PR adds a pulse rifle to the system, so the boots will be irrelevant.

You can merge this no problem.

**Pull request merged**